### ISSUE TITLE
Vulcan next & MERN

### ISSUE THREAD

**Issue by j209** [VulcanJS/vulcan-next#161]
Hi
How we can compare vulcan-next and MERN Stack?
Can we say vulcan a  is a successor and mature of MERN?

**Comment by eric-burel** [comment]
Hi, good question, let me try an elaborate response:

### TL;DR

- MERN stack is set of technologies that helps creating full-stack web application, perfectly fine to use at scale but also the way you'll probably learn web development at school. Doesn't aim at solving business issue, it's more a technical basis.
- Vulcan, Redwood, Blitz, Meteor: more targeted towards creating SaaS software, in a business/startup context. Meant to create business value quickly and also in the long run.
- Remix, Next, CRA, Vite: backbones for well-engineered "center-stack" applications, leaning on the frontend or "backend for frontend" aspect of web dev. Meant for creating applications that are mosty UIs, or to serve as the basis for more complex applications. Vulcan and Blitz are both built on top of Next for example. 

### MERN stack: the way to start
In my opinion, the MERN stack, Mongo Express React and Node, is roughly what should be taught at school when learning modern web development.
I am not that adamant on the Mongo part though, but it's easier to learn for people without a strong CS background, thanks to its schema-less approach. Also for React, you can of course replace it with Angular to get the MEAN stack or Vue or Svelte.

I learnt web development with the MEAN stack and was happy with it.

### Vulcan, Redwood, Meteor, Blitz: opinionated solutions to a variety of business issues

Now, I wouldn't call anything "successor" to the MERN stack. The MERN stack is perfectly mature, and a perfectly fine way to create full-stack applications. 

The reason for a framework like Vulcan to exist is that the MERN stack is really low-level. It doesn't explain how you should structure your API, how to avoid code duplication, how to make it more reusable across technologies and so many small issues you'll encounter in a real-life application.

Imo, they operate more at the business level, meaning that they bring features that creates value in a business setting, typically user authentication. The MERN stack is way less opinionated, it's operating on the technical level.

### CRA, Vite, Remix, Next: backbones for full-stack development with advanced frontends

Then you have a new kind of framework, that operates at an even different level. They can be considered as backbones to frontend applications.
It's kind a of a shift from full-stack to "center-stack", those frameworks solves problems typically encountered by frontend architect. They don't necessarily claim to be the solution for advanced backend development, but they provide enough features for most apps.

**Comment by j209** [comment]
Thank you very much for your elaborate response.
